sdk/include/opentelemetry/sdk/trace/span_data.h

```diff
@@ -107,9 +41,41 @@ class SpanDataEvent
    */
   core::SystemTimestamp GetTimestamp() const noexcept { return timestamp_; }
 
+  /**
+   * Get the attributes for this event
+   * @return the attributes for this event
+   */
+  const std::unordered_map<std::string, SpanDataAttributeValue> &GetAttributes() const noexcept
+  {
+    return attribute_map_.GetAttributes();
+  }
+
 private:
   std::string name_;
   core::SystemTimestamp timestamp_;
+  AttributeMap attribute_map_;
+};
+
+/**
+ * Class for storing links in SpanData.
+ * TODO: Add trace_id, span_id and trace_state when these are supported by SpanContext
```

Comment:
Can't we just add the `SpanContext` as it is? At some point it will contain the ids and the trace state, then this code doesn't need to updated.

Reply:
Sure! I added a `SpanContext` to `SpanData`. I think once the ids and trace state are supported in `SpanContext`, we can add getters for them here.